ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -45,12 +43,6 @@ message GetActiveContractsRequest {
   // In particular, setting the verbose flag to true triggers the ledger to include labels for record fields.
   // Optional
   bool verbose = 3;
-
-  // Server side tracing will be registered as a child of the submitted context.
-  // This field is a future extension point and is currently not supported.
-  // Optional
-  TraceContext trace_context = 1000;
```

Comment:
Perhaps 1000 should be reserved across this PR; maybe it doesn't matter.

Reply:
Indeed, we could make these fields reserved, however, as we aren't running different versions of the participant server against the same application this shouldn't matter (i.e., even if the application specified a field with ID 1000 it'll be ignored at all times).